src/presentation/cli/controller/account.go

```diff
@@ -21,14 +21,80 @@ func NewAccountController(
 }
 
```

Comment:
Hey, you created a CLI endpoint for that. It wasn't needed, but guess I should have mentioned on the task. Sorry, my bad. I think that if someone has CLI access, they can add public keys on .authorized_keys manually. But that's ok, we save them on the database for rapid retrieval, so I guess it makes sense to use the CLI? I think nobody will tho.

Reply:
I believe this is a classic case of educating the user to use things as they were designed. Adding a key directly to the `authorized_keys` file will prevent them from managing that key through the interface and API. It's similar to what we did with the creation of `DONOTDELETE` files, there’s not much room to maneuver, but educating the user on how to use this feature is worthwhile.